apps/rxjs.dev/src/app/custom-elements/operator-decision-tree/README.md

```diff
@@ -1,8 +1,9 @@
 The `operator-decision-tree` module requires `decision-tree-data.json`, which is hosted at `/generated/app`.
 
-The JSON is generated via the `docs_app/tools/decision-tree-generator`.
+The JSON is generated via the `apps/rxjs.dev/tools/decision-tree-generator`.
 
 # TODO
+
 - Consider placing the widget on the home page - or a link on the home page, “Decision Tree”
 - Manual focus calls when navigating the tree (example: after making a selection, focus on the current sentence)
-- Drop jasmine-marbles for just the TestScheduler
\ No newline at end of file
+- Drop jasmine-marbles for just the TestScheduler
```

Comment:
I think I dropped this in #7238. Or at least I wanted to. Whatever it is, I will have to think how to fix that PR to work with these changes.

Reply:
There shouldn't be any real changes in this PR, it's just moving files and updating build scripts.